scripts/Palindrome.py

```diff
@@ -1,41 +1,40 @@
+IGNORE_CHAR_LIST = ' .,-_:!@#$%^&?'
+
+
 def is_palindrome(sentence):
```

Comment:
Better to name "text" to be more generic (and shorter as an added benefit)
We can create a palindrome text from several sentences